# Ensure the types.d.ts can be parsed by api-extractor
diff --git a/types.d.ts b/types.d.ts
index f58f6dfe6a2c2098385d1ccf33200baeb31b2166..b377c331e0684914bef1726ad41b42107ed574ee 100644
--- a/types.d.ts
+++ b/types.d.ts
@@ -271,7 +271,6 @@ declare interface StreamChunksOptions {
 	finalSource?: boolean;
 	columns?: boolean;
 }
-declare namespace exports {
 	export namespace util {
 		export namespace stringBufferUtils {
 			export let disableDualStringBufferCaching: () => void;
@@ -320,6 +319,4 @@ declare namespace exports {
 		GeneratedSourceInfo,
 		StreamChunksOptions
 	};
-}
 
-export = exports;
